html/pfappserver/lib/pfappserver/Form/Config/Profile.pm

```diff
@@ -69,6 +69,25 @@ has_field 'filter.contains' =>
    widget_wrapper => 'DynamicTableRow',
   );
 
+=head2 filter_match_style
+
+The field for filter_match_style
```

Comment:
Shouldn't that be "more descriptive" :P
I know that it is pretty obvious, but still...